util/internal/integrationtest/common/src/main/java/org/hibernate/search/util/impl/integrationtest/common/rule/BackendMock.java

```diff
@@ -189,14 +192,11 @@ public DocumentWorkCallListContext expectWorks(String indexName, String tenantId
 		);
 	}
 
-	public IndexScaleWorkCallListContext expectIndexScaleWorks(String indexName) {
-		return expectIndexScaleWorks( indexName , null );
-	}
-
-	public IndexScaleWorkCallListContext expectIndexScaleWorks(String indexName, String tenantId) {
+	public IndexScaleWorkCallListContext expectIndexScaleWorks(String indexName, String... tenantIds) {
 		CallQueue<IndexScaleWorkCall> callQueue = backendBehavior().getIndexScaleWorkCalls( indexName );
 		return new IndexScaleWorkCallListContext(
-				indexName, tenantId,
+				indexName,
+				Arrays.asList( tenantIds ).stream().filter( Objects::nonNull ).collect( Collectors.toList() ),
```

Comment:
Better use a `Set` here, so that order is irrelevant. That would require using a `Set` in the mappers as well, though (as I suggested elsewhere).